config/default.js

```diff
@@ -187,6 +187,7 @@ module.exports = {
         process.env.CHALLENGER_KEY ||
         process.env.BOOT_CHALLENGER_KEY ||
         '0xd42905d0582c476c4b74757be6576ec323d715a0c7dcff231b6348b7ab0190eb',
```

Comment:
Why is it defined as part of the environment? If you maintain it there, can you also add an entry in aws?

Reply:
Yes, let's add it to aws and polygonEdge!